Gulpfile.mjs

```diff
@@ -91,11 +93,12 @@ if (process.env.CIRCLE_PR_NUMBER) {
   versionSuffix = "+pr." + process.env.CIRCLE_PR_NUMBER;
 }
 
-const babelVersion =
-  require("./packages/babel-core/package.json").version + versionSuffix;
-function buildRollup(packages) {
+const babelVersionNumber = await import("./packages/babel-core/package.json")
+  .version;
+const babelVersion = babelVersionNumber + versionSuffix;
+async function buildRollup(packages) {
   const sourcemap = process.env.NODE_ENV === "production";
-  return Promise.all(
+  return await Promise.all(
```

Comment:
I don't think this function has to be marked as async, unless `buildRollup` has been `await`ed somewhere.